### ISSUE TITLE
PipeNet Atmos Invalid Grid Exception

### ISSUE THREAD

**Issue by Acruid** [space-wizards/space-station-14#3820]
Showing up regularly in Grafana, no other info is known. I'm assuming pipenet isn't getting cleared between round restarts?
```
System.Collections.Generic.KeyNotFoundException: The given key '16' was not present in the dictionary.
at System.Collections.Generic.Dictionary`2.get_Item(TKey key) in /_/src/libraries/System.Private.CoreLib/src/System/Collections/Generic/Dictionary.cs:line 181
at Robust.Shared.Map.MapManager.GetGrid(GridId gridID) in /home/runner/work/space-station-14/space-station-14/RobustToolbox/Robust.Shared/Map/MapManager.cs:line 501
at Content.Server.GameObjects.EntitySystems.AtmosphereSystem.GetGridAtmosphere(GridId gridId) in /home/runner/work/space-station-14/space-station-14/Content.Server/GameObjects/EntitySystems/AtmosphereSystem.cs:line 137
at Content.Server.GameObjects.Components.NodeContainer.NodeGroups.PipeNet.AfterRemake(IEnumerable`1 newGroups) in /home/runner/work/space-station-14/space-station-14/Content.Server/GameObjects/Components/NodeContainer/NodeGroups/IPipeNet.cs:line 88
at Content.Server.GameObjects.Components.NodeContainer.NodeGroups.BaseNodeGroup.RemakeGroup() in /home/runner/work/space-station-14/space-station-14/Content.Server/GameObjects/Components/NodeContainer/NodeGroups/INodeGroup.cs:line 101
at Content.Server.GameObjects.Components.NodeContainer.NodeGroups.NodeGroupManager.Update(Single frameTime) in /home/runner/work/space-station-14/space-station-14/Content.Server/GameObjects/Components/NodeContainer/NodeGroups/INodeGroupManager.cs:line 31
at Robust.Shared.GameObjects.EntitySystemManager.TickUpdate(Single frameTime) in /home/runner/work/space-station-14/space-station-14/RobustToolbox/Robust.Shared/GameObjects/EntitySystemManager.cs:line 266
```

**Comment by Zumorica** [comment]
Will look into it as I'm cleaning up/reworking pipes and pipenets

**Comment by Zumorica** [comment]
Fixed by #3833
